llvm/include/llvm/Support/TimeProfilerTool.h

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines command line options and helper class for tools that want
+// to use time trace profiling.
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_SUPPORT_TIMEROFILER_TOOL_H
+#define LLVM_SUPPORT_TIMEROFILER_TOOL_H
+
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/TimeProfiler.h"
+
+namespace llvm {
```

Comment:
The header is intended to be included in the a .cpp file directly, so I can also make this anonymous and rename it to .inc to indicate its not a regular header to be included from multiple files.